Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -507,6 +510,9 @@ RefPtr<CSSCustomPropertyValue> CSSPropertyParser::parseTypedCustomPropertyValue(
         case CSSCustomPropertySyntax::Type::CustomIdent:
             return { downcast<CSSPrimitiveValue>(value).stringValue() };
 
+        case CSSCustomPropertySyntax::Type::String:
+            return { serializeString(downcast<CSSPrimitiveValue>(value).stringValue()) };
+
```

Comment:
Why the blank lines?

Reply:
Removed, we were sort of being inconsistent about it before.